### PR TITLE
ZFIN-8356: Updated nodejs packages to fix vulnerabilities.

### PR DESCRIPTION
**Pull request by zfinryanm**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,51 +16,51 @@
...
+    "jquery-modal": "0.9.2",
     "jquery-stickytabs": "1.2.4",
-    "jquery-validation": "^1.19.3",
+    "jquery-validation": "^1.19.5",
     "jquery.tipsy": "1.0.3",
     "masonry-layout": "^4.2.2",
-    "mini-css-extract-plugin": "^1.6.2",
-    "node-sass": "^5.0.0",
+    "mini-css-extract-plugin": "^2.7.2",
+    "node-sass": "^6.0.1",
```
Comment (rtaylorzfin) [inline package.json:47]: Is it too early to switch to "sass"?

**Review by rtaylorzfin — APPROVED** [review]
Looks good to me. I wonder if we could even add the "sass" package to replace "node-sass".  If you think that's too crazy to do right now, we can wait on that one.

File: package.json (hunk 1 above)
Comment (zfinryanm) [reply package.json:47]: I'd rather separate out the switch so any problems from it aren't mixed in with these upgrades.

**Comment by rtaylorzfin** [comment]
Makes sense

From: zfinryanm ***@***.***>
Date: Monday, January 23, 2023 at 4:14 PM
To: ZFIN/zfin ***@***.***>
Cc: Ryan Taylor ***@***.***>, Review requested ***@***.***>
Subject: Re: [ZFIN/zfin] ZFIN-8356: Updated nodejs packages to fix vulnerabilities. (PR #494)

@zfinryanm commented on this pull request.

________________________________

In package.json<https://github.com/ZFIN/zfin/pull/494#discussion_r1084700168>:

>      "jquery.tipsy": "1.0.3",

     "masonry-layout": "^4.2.2",

-    "mini-css-extract-plugin": "^1.6.2",

-    "node-sass": "^5.0.0",

+    "mini-css-extract-plugin": "^2.7.2",

+    "node-sass": "^6.0.1",

I'd rather separate out the switch so any problems from it aren't mixed in with these upgrades.

**Pull request merged**